eth/backend.go

```diff
@@ -115,6 +117,7 @@ func roundUpCacheSize(input int, allocSize int) int {
 // New creates a new Ethereum object (including the
 // initialisation of the common Ethereum object)
 func New(
+	ctx *snow.Context,
```

Comment:
can we remove this parameter in favor of setting the `snow.Context` on the AvalancheContext prior to initializing the ethereum backend?